ci/Dockerfile

```diff
@@ -1,62 +1,140 @@
-FROM a12e/docker-qt:5.14-gcc_64
+ARG QT_VERSION=5.15.2
+
+# QT Installation Image --------------------------------------------------------
+FROM ubuntu:18.04 AS qt-install
+
+ARG QT_VERSION
+
+RUN apt update && apt full-upgrade -y && apt install -y --no-install-recommends \
+    sudo python3 python3-pip \
+    && apt-get -qq clean
+
+RUN python3 -m pip install setuptools
+
+RUN chmod -R 777 /opt
+
+RUN groupadd -r user && useradd --create-home --gid user user && echo 'user ALL=NOPASSWD: ALL' > /etc/sudoers.d/user
+
+
+COPY scripts/install-qt.sh /home/user/
+RUN chown user:user /home/user/install-qt.sh
+RUN chmod a+x /home/user/install-qt.sh
+
+USER user
+WORKDIR /home/user
+
+ENV HOME /home/user
+ENV QT_VERSION=$QT_VERSION
+
+# Install QT
+RUN /home/user/install-qt.sh
+
+
+
+# Build Image ------------------------------------------------------------------
+FROM ubuntu:18.04
+
+ARG LINUXDEPLOYQT_VERSION=continuous
+ARG QT_VERSION
+
+
+# Adapted from a12e/docker-qt by Aurélien Brooke
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    QMAKESPEC=linux-g++ \
+    QT_PATH=/opt/qt \
+    QT_PLATFORM=gcc_64 \
+    QT_VERSION=$QT_VERSION
+
+ENV \
+    PATH=${QT_PATH}/${QT_VERSION}/${QT_PLATFORM}/bin:$PATH
```

Comment:
Why is this line broken like that? And not part of the previous `ENV` directive?